.circleci/config.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           command: yarn build
       - run:
           name: Yarn Prod Deploy
-          command: aws s3 sync build/ s3://semgrep-docs/docs
+          command: aws s3 sync build/ s3://semgrep-docs/docs --delete
```

Comment:
I'm guessing this would only need to be run once, and then we can go back to the original, non-delete version?

Reply:
Adding this flag causes `aws s3` to look for local files that are no longer present on the destination, so we’d want to keep it in moving forward. That’ll prevent stale files from remaining in the destination indefinitely.